openmc/mgxs/groups.py

```diff
@@ -299,3 +303,127 @@ def merge(self, other):
         # Assign merged edges to merged groups
         merged_groups.group_edges = list(merged_edges)
         return merged_groups
+
+
+class DelayedGroups(object):
+    """A delayed groups structure used for multi-delayed-group parameters.
+
+    Parameters
+    ----------
+    groups : Iterable of Int
+        The delayed groups
```

Comment:
Perhaps it is just me, but I'm not clear on what this would actually be. If my cross section library has 6 delayed groups, then this allows me to specify some list - e.g., `[1,2,4,6]` - of the groups that I want to tally?